python/tank/authentication/sso_saml2/core/sso_saml2_core.py

```diff
@@ -138,6 +148,19 @@ class TKWebPage(QtWebKit.QWebPage):
              - Any other links which may be presented by SSO Providers
             """
 
+            def __init__(self, parent=None):
+                """
+                Class Constructor.
+                """
+                get_logger().debug('TKWebPage.__init__')
+                super(TKWebPage, self).__init__(parent)
+
+            def __del__(self):
+                """
+                Class Destructor.
+                """
+                get_logger().debug('TKWebPage.__del__')
```

Comment:
probably not necessary, but you should call the base classes' `__del__`

Reply:
Well... when I call the destructor, I do get a:
`Exception AttributeError: "'super' object has no attribute '__del__'" in <bound method TKWebPage.__del__ of <tank.authentication.sso_saml2.core.sso_saml2_core.TKWebPage object at 0x10c7b20e0>> ignored
`

An exception thrown in a dtor is not a good thing as it stops the processing of the other dtor in the hierarchy.